src/pudl/metadata/resources/ferc1.py

```diff
@@ -32,6 +32,7 @@
                 "energy_source_type",
                 "row_type_xbrl",
                 "energy_mwh",
+                "record_id",
```

Comment:
Did these tables not have `record_id` columns for some reason?

Reply:
whooops!

Reply:
🙈